package.json

```diff
@@ -4,7 +4,8 @@
   "homepage": "https://github.com/lloyd/node-toobusy",
   "version": "0.2.4",
   "dependencies": {
-    "bindings": "1.1.0"
+    "bindings": "1.1.0",
+    "nan": "1.6.2"
```

Comment:
Can you update this to nan v1.8.4? That would allow toobusy to install on io.js.